src/compiler.jl

```diff
@@ -450,13 +454,12 @@ function emit_allocobj!(B, tag::LLVM.Value, Size::LLVM.Value, needs_workaround::
     curent_bb = position(B)
     fn = LLVM.parent(curent_bb)
     mod = LLVM.parent(fn)
-	ctx = context(mod)
 
-	T_jlvalue = LLVM.StructType(LLVMType[]; ctx)
+	T_jlvalue = LLVM.StructType(LLVMType[])
```

Comment:
Formatting

Reply:
Isn't the diff here on the removal of ctx rather than whitespace? I've otherwise gone through whitespace diffs and tried to get rid of them apart from a couple of weird indents that seemed out of place.